### PR TITLE
MRG: Fix hpi filtering after MF downsample

### PR DESCRIPTION
**Pull request by Eric89GXL**
Fixes a bug with `chpi` filtering after MaxFilter's downsampling (and ours, even, really).

Ready for review/merge from my end. Hopefully the test comments and code make it clear why this is necessary.

### CODE REVIEW

**Comment by Eric89GXL** [comment]
cc @mdclarke this should fix the bug you hit

File: mne/io/base.py [hunk 1]
```diff
@@ -1065,6 +1065,8 @@ def resample(self, sfreq, npad=None, window='boxcar', stim_picks=None,
 
         inst._data = np.concatenate(new_data, axis=1)
         inst.info['sfreq'] = sfreq
+        if inst.info.get('lowpass') is not None:
+            inst.info['lowpass'] = min(inst.info['lowpass'], sfreq / 2.)
```
Comment (Eric89GXL) [inline mne/io/base.py]: Turns out my changes uncovered a bug with `raw.resample()` where we didn't update `raw.info['lowpass']`, and we should have.

**Comment by Eric89GXL** [comment]
Can we abandon Python2.6 yet? :(

**Comment by agramfort** [comment]
+1 for merge when travis is happy

**Comment by Eric89GXL** [comment]
Do we need 2.6 support still? numpy.testing is missing a function there

**Comment by Eric89GXL** [comment]
And don't forget (
https://github.com/mne-tools/mne-python/issues/2759#issuecomment-168428421)
:) In this case it's probably old numpy actually, but I'd have to check. It
might be too recent a version to drop.

**Comment by agramfort** [comment]
yaya ... tell me if it's old numpy. I suspect it is

**Comment by Eric89GXL** [comment]
`assert_raises_regex` (the missing function) is from only a year ago so I will backport

**Comment by Eric89GXL** [comment]
@agramfort I'm removing `fixes.py` from our coverage, since I don't really think it makes sense to have it there as it's really just duplicating code from other repos like `externals` does

**Comment by Eric89GXL** [comment]
Okay backported, @agramfort let me know if you're okay with the backport and omitting `fixes.py` from coverage and I'll merge (or you can do it)

**Comment by agramfort** [comment]
Thx

**Pull request merged**